scripts/mdpow-pcw

```diff
@@ -183,61 +183,71 @@ def realpath_or_none(fn):
-                      "determine the file type. 'auto' generates a pdf file "
-                      "DIRECTORY/dVdl_MOLID_pcw.pdf and 'None' disables it [%default]"
-                      "The plot function is only available for mdpow estimator,"
-                      "and is disabled when using alchemlyb estimators.",
-                      metavar="FILE")
-    parser.add_option('-o', '--outfile', dest="outfile",
-                      help="append one-line results summary to FILE [%default]",
-                      metavar="FILE")
-    parser.add_option('-e', '--energies', dest="energyfile",
-                      help="append solvation free energies to FILE [%default]",
-                      metavar="FILE")
-    parser.add_option('--estimator', dest="estimator", default='alchemlyb',
-                      help="choose the estimator to be used, alchemlyb or mdpow estimators")
-    parser.add_option('--method', dest="method", default='MBAR',
-                      help="choose the method to calculate free energy",)
-    parser.add_option('--force', dest='force',
-                      action="store_true",
-                      help="force rereading all data [%default]")
-    parser.add_option('--noSI', dest='SI',
-                      action="store_false",
-                      help="Disable statistical inefficiency analysis."
-                      "Statitical inefficiency analysis is performed by default when using"
-                      "alchemlyb estimators and is disabled when using mdpow estimator.")
-    parser.add_option('-s', '--stride', dest="stride", type='int',
-                      help="Use every N-th datapoint from the original dV/dlambda data. "
-                      "[%default]",
-                      metavar="N")
-    parser.add_option('--start', dest="start", type='int',
-                      help="Start point for the data used from the original dV/dlambda data. ")
-    parser.add_option('--stop', dest="stop", type='int',
-                      help="Stop point for the data used from the original dV/dlambda data.",)
-    parser.add_option('--ignore-corrupted', dest="permissive",
-                      action="store_true",
-                      help="skip lines in the md.xvg files that cannot be parsed. "
-                      "WARNING: Other lines in the file might have been corrupted in "
-                      "such a way that they appear correct but are in fact wrong. "
-                      "WRONG RESULTS CAN OCCUR! USE AT YOUR OWN RISK [%default]")
-    parser.set_defaults(plotfile="none",
-                        outfile="pcw.txt", energyfile="energies.txt",
-                        force=False, stride=1, permissive=False, SI=True)
-    opts,args = parser.parse_args()
-
-    if len(args) == 0:
-        logger.fatal("A directory is required. See --help.")
-        sys.exit(1)
-    elif len(args) > 1 and not opts.plotfile.lower() in ('none', 'auto'):
+    import argparse
+
```

Comment:
I am slightly confused why this shows up in your diff. Maybe not everything from develop is merged?

Reply:
Oh forgot to update that file to the latest my most recent commit fixed that

Reply:
Your are right I didn't merge all of it, fixed in later commit.

Reply:
Didn't completely merge this file, fixed in fc90717